packages/portal-proto/src/components/ClearStoreErrorBoundary.tsx

```diff
@@ -25,7 +25,7 @@ class ClearStoreErrorBoundary extends Component<
             An unexpected error has occurred. Your saved cohorts are being
             recovered, but your cart and sets may be lost.
           </p>
-          <ModalButtonContainer>
+          <ModalButtonContainer data-testid="modal-button-container">
```

Comment:
If it's a button, this should start with `button-`

Reply:
As discussed it is not a button, but rather the bottom portion of the modal that contains buttons